internal/schema/group.go

```diff
@@ -22,6 +22,7 @@ type Group struct {
 		User User          `json:"user"`
 		Role api.GroupRole `json:"role"`
 	} `json:"members,omitempty"`
+	Groups []Group `json:"groups,omitempty"`
```

Comment:
I might have missed it, but does this and the other `Groups` field in the `User` struct get used anywhere?

Reply:
Well this PR went downhill real fast... Forgot I had these in here for another thing I was working on.